### ISSUE TITLE
[Visual bug] Misaligned image layers

### ISSUE THREAD

**Issue by Ouaz** [bvschaik/julius#720]
Nitpicky visual bugs (happens on vanilla C3 too): image layers of working buildings are slightly shifted compared to the same buildings when inactive.

- Amphitheater
ID 3050 is 1 pixel too far to the left (compared to ID 3039)
![amphi_on_off](https://github.com/bvschaik/julius/assets/11946570/5af3b9a2-264f-4f4e-ab76-d3faf06e1d82)

- Lion Pit : 
ID 3101 to ID 3118 are 1 pixel to low (compared to ID 3100)
![pit_on_off](https://github.com/bvschaik/julius/assets/11946570/6467dc3f-92ee-43b7-848b-f4f8f794603c)

To reproduce, simply switch between Global Entertainment overlay and City map.

------

- Engineer's post
ID 3282 to 3291 are 1 pixel too high (compared to ID 3281)
![engineer_post_on_off](https://github.com/bvschaik/julius/assets/11946570/c451c91b-76a1-417b-91e1-d526f0f5b025)
This one has been fixed a long time ago in Augustus (April 2022), I forgot to report it first for Julius.

**Comment by crudelios** [comment]
I think the Theater is also misaligned in Julius, can you confirm please?

**Comment by bvschaik** [comment]
The problem with fixing these is that the offsets are inside the SG2 files. Not sure how Augustus fixed those -- by hardcoding image ID + x/y offset in the code?

**Comment by Ouaz** [comment]
> Not sure how Augustus fixed those -- by hardcoding image ID + x/y offset in the code?

Found the commit that fixed the roof of active Engineer's post: https://github.com/Keriew/augustus/commit/2923909686013204dee49d33efd3f8ebc1138ab8
Amphitheater and Lion Pit have not been fixed.

> I think the Theater is also misaligned in Julius, can you confirm please?

I see nothing wrong with the Theater. All's good.

**Comment by crudelios** [comment]
> The problem with fixing these is that the offsets are inside the SG2 files. Not sure how Augustus fixed those -- by hardcoding image ID + x/y offset in the code?

Yes the changes were hard coded. Not the best solution but it works!

**Comment by crudelios** [comment]
Should we fix this? (doing it manually on `core/image.c` or do I keep this to Augustus?

**Comment by bvschaik** [comment]
Tough one. It does look better if fixed, even though the fix itself is quite ugly... Let's go ahead and fix.
